api/Navigator.json

```diff
@@ -3754,6 +3755,11 @@
                     "name": "dom.webmidi.enabled",
                     "value_to_set": "true"
                   }
+                ],
+                "notes": [
+                  "Version 107 and later, API use is gated by a site permission add-on that is automatically generated by Firefox.",
+                  "Version 105 and later, the <code>dom.webmidi.gated</code> preference may be set <code>false</code> to allow WebMIDI to be used without a site permission addon.",
+                  "Version 100 to 104 (inclusive) WebMIDI is not available (API use is gated by a site permission add-on that is no longer available)."
                 ]
```

Comment:
The preference version includes all the data. I am expecting this to be deleted when the feature goes live.